onnxruntime/python/tools/quantization/E2E_example_model/CalTableFlatBuffers/TrtTable.py

```diff
@@ -0,0 +1,51 @@
+# automatically generated by the FlatBuffers compiler, do not modify
```

Comment:
why is this under E2E_example_model ?

Reply:
move CalTableFlatBuffers to quantization tool root folder.